src/services/useDisplayConfig/index.ts

```diff
@@ -41,16 +42,34 @@ function isTimeSeriesGridActionsFilter(
 function actionsResponseToDisplayConfig(
   actionsResponse: ActionsResponse,
   nodeId: string | undefined,
+  startTime?: Date,
+  endTime?: Date,
 ): DisplayConfig[] {
   const displays: DisplayConfig[] = []
   for (const result of actionsResponse.results) {
     if (result.config === undefined) continue
     const title = result.config.timeSeriesDisplay.title ?? ''
     const timeSeriesDisplayIndex = result.config.timeSeriesDisplay.index
     const period = result.config.timeSeriesDisplay.period
+
+    // The period is always specified in UTC.
+    const timeZoneOffsetString = 'Z'
+    let configPeriod: [Date, Date]
+    if (period) {
+      const periodStart = convertFewsPiDateTimeToJsDate(
+        period.startDate,
+        timeZoneOffsetString,
+      )
+      const periodEnd = convertFewsPiDateTimeToJsDate(
+        period.endDate,
+        timeZoneOffsetString,
+      )
+      configPeriod = [startTime ?? periodStart, endTime ?? periodEnd]
```

Comment:
We don't need to convert the FEWS PI date/times if we already know we're not going to use them?